tests/scripts/thread-cert/node.py

```diff
@@ -432,6 +474,11 @@ def set_panid(self, panid=config.PANID):
         self.send_command(cmd)
         self._expect('Done')
 
+    def set_parent_priority(self, priority):
+        cmd = 'parentpriority %d' % priority
+        self.send_command(cmd)
```

Comment:
```suggestion
        self.send_command('parentpriority %d' % priority)
```

Reply:
@edmont here would prefer keep it as is to stay the same style as the others.

Reply:
@librasungirl it seems there are [mixed styles](https://github.com/openthread/openthread/blob/da4e64805219e695962b96f570244907e72f0b4a/tests/scripts/thread-cert/node.py#L504). But it's okay, not big deal.

Reply:
Noted: Another one to enhance! Thanks!